src/main.rs

```diff
@@ -84,10 +84,14 @@ async fn serenity(#[shuttle_runtime::Secrets] secret_store: SecretStore) -> Shut
 					poise::Prefix::Literal("🦀"),
 					poise::Prefix::Literal("<:ferris:358652670585733120> "),
 					poise::Prefix::Literal("<:ferris:358652670585733120>"),
-					poise::Prefix::Literal("<:ferrisballSweat:678714352450142239> "),
-					poise::Prefix::Literal("<:ferrisballSweat:678714352450142239>"),
+					poise::Prefix::Literal("<:sweat:678714352450142239> "),
+					poise::Prefix::Literal("<:sweat:678714352450142239>"),
+					poise::Prefix::Literal("<:owo:678714352450142239> "),
+					poise::Prefix::Literal("<:owo:678714352450142239>"),
+					poise::Prefix::Literal("<:owo:579331467000283136> "),
+					poise::Prefix::Literal("<:owo:579331467000283136>"),
```

Comment:
what's the prefixes change?

Reply:
![image](https://github.com/rust-community-discord/ferrisbot-for-discord/assets/70787919/e07f9aa5-fe3d-41ca-bbc6-d3caec5d23af) apply `:#?` formatting